### PR TITLE
Remove cluster logging step to get channel version

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1784929

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill @bparees PTAL. 

I will update 4.3 with the 4.3 channel after merge.

File: modules/cluster-logging-deploy-subscription.adoc [hunk 1]
```diff
@@ -154,7 +146,7 @@ spec:
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
-<2> Specify the `.status.channels[].name` value from the previous step.
+<2> Specify channel version `4.2`.
```
Comment (bparees) [inline modules/cluster-logging-deploy-subscription.adoc]: so we're going to have to keep updating this doc w/ every release?  can we somehow make this more generic so we don't have that problem?

Comment (mburke5678) [reply modules/cluster-logging-deploy-subscription.adoc]: Is there a plan to get the Elasticserch Operator to install using the web console? If users could install via the web console all of these steps become unnecessary.

Comment (bparees) [reply modules/cluster-logging-deploy-subscription.adoc]: i thought there was some plan to have cluster logging installation automatically pull in elastic via a dependency process.

Comment (jcantrill) [reply modules/cluster-logging-deploy-subscription.adoc]: afaik its possible to do everything from the console though not necessarily through the OLM bits.

**Pull request merged**